src/components/panel/panel.js

```diff
@@ -464,19 +484,34 @@ function MdPanelRef(config, $injector) {
  * is opened and animations finish.
  */
 MdPanelRef.prototype.open = function() {
-  if (!this.isOpen) {
-    this.isOpen = true;
+  if (this.isOpen) {
+    // Ignore open. We don't want chaining to happen here.
+    return this._$q.reject('Panel is already opened.');
```

Comment:
This was a design decision, but are discussing it more, we combined this if case with the next one and removed the finally.